src/nomad_simulations/properties/energies.py

```diff
@@ -77,3 +78,502 @@ def __init__(
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+class ElectrostaticLongRangeEnergy(ClassicalEnergy):
+    """
+    Physical property section describing long-range electrostatic contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class IntramolecularEnergy(ClassicalEnergy):
+    """
+    Physical property section describing all intramolecular contributions to the potential energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class BondEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from bond interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class AngleEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from angle interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class DihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ImproperDihedralEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from improper dihedral interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+
+class ExternalEnergy(ClassicalEnergy):
+    """
+    Physical property section describing contributions to the potential energy from external interactions of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+
+######################################
+# List of quantum energy contributions
+######################################
+
+class ElectronicEnergy(QuantumEnergy):
+    """
+    Physical property section describing the electronic energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
+#! allowed_contributions = ['PotKin'] Not sure how to deal with sub-contributions... - I lean towards keeping a flat list but I am not sure of all the usages
+
+
+class ElectronicKineticEnergy(QuantumEnergy):
+    """
+    Physical property section describing the electronic kinetic energy of a (sub)system.
+    """
+
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
```

Comment:
Okay, this is an interesting take. In Solid State Physics, the energies below also typically refer to electrons, but technically may be defined for any part of the system.
I would think of a way for distinguishing the kind of energy contribution from the system it applies to. That will keep the wildgrowth manageable.